sdk/security/keyvault/azkeys/internal/transforms.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package main
+
+import (
+	"log"
+	"os"
+	"regexp"
+)
+
+// removing client prefix from types
+func regexReplace(fileName string, regex string, replace string) {
+	r, err := regexp.Compile(regex)
+	if err != nil {
+		panic(err)
+	}
+
+	file, err := os.ReadFile(fileName)
+	if err != nil {
+		panic(err)
+	}
+
+	file = r.ReplaceAll(file, []byte(replace))
+
+	err = os.WriteFile(fileName, file, 0644)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func main() {
+	// delete the version path param check (version == "" is legal for Key Vault but indescribable by OpenAPI)
+	regexReplace("client.go", `\sif version == "" \{\s+.+version cannot be empty"\)\s+\}\s`, "")
```

Comment:
And also indescribable by TypeSpec?

Reply:
Apparently not as the compiler rejects it.
```
-path-param: Path parameter 'keyVersion' cannot be optional.
> 201 |     keyVersion?: string;
      |     ^^^^^^^^^^

Found 1 error.
```